### PR TITLE
Add metrcis for PortingAssistantCLI

### PR DESCRIPTION
**Pull request by flywanli**
#### Description of change
[//]: # (What are you trying to fix? What did you change)
#140 
#### Issue
[//]: # (Having an issue # for the PR is required for tracking purposes. If an existing issue does not exist please create one.)

#### PR reviewer notes
[//]: # (Let us know if there is anything we should focus on.)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj [hunk 1]
```diff
@@ -1,41 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
```
Comment (ShanshanHe) [inline src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj:3]: Can you change the tab to space and remove unnecessary changes please.

Comment (flywanli) [reply src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj:3]: my vs code auto format like this. it should be fine

File: .github/workflows/build_test.yml [hunk 2]
```diff
@@ -17,10 +17,14 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - name: Setup .NET Core
+    - name: Setup .NET Core 3
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 3.1.301
+        dotnet-version: 3.1.412
```
Comment (ShanshanHe) [inline .github/workflows/build_test.yml:23]: If we are using dotnet 5, do we even need dotnet 3 here?

File: src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj (hunk 1 above)
Comment (ShanshanHe) [reply src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj:3]: no it's not fine, all the unnecessary changes creates a lot of merge conflict for someone who is currently working on the same file. We should keep our commit change as clean and tide as possible.

**Review by ShanshanHe — CHANGES_REQUESTED** [review]

File: src/PortingAssistant.Client.Telemetry/Model/APIMetrics.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
```
Comment (ShanshanHe) [inline src/PortingAssistant.Client.Telemetry/Model/APIMetrics.cs:1]: Something to consider since we have added all the telemetry data structure here: in the future we should update [`porting-assistant-dotnet-ui`](https://github.com/aws/porting-assistant-dotnet-ui/tree/develop/packages/csharp/PortingAssistant/PortingAssistant.Telemetry) and [PortingAssistantCLI](https://code.amazon.com/packages/PortingAssistantCLI/trees/mainline/--/PortingAssistantCLI/Model) to use the ones from `PortingAssistant.Client.Telemetry` nuget package. This way we keep our telemetry data consistent, and reduce maintenance overhead. Maybe we should add a ticket/issue to keep track of this.

**Review by ShanshanHe — APPROVED** [review]

File: src/PortingAssistant.Client.Telemetry/Model/APIMetrics.cs (hunk 3 above)
Comment (flywanli) [reply src/PortingAssistant.Client.Telemetry/Model/APIMetrics.cs:1]: This is AI I already told anrudda, he will work on this once he finished his current work

File: src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj [hunk 4]
```diff
@@ -5,30 +5,25 @@
...
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0" />
     <PackageReference Update="Nerdbank.GitVersioning" Version="3.4.231" />
   </ItemGroup>
-
+  
+  <ItemGroup>
+    <Folder Include="NewFolder\" />
```
Comment (ShanshanHe) [inline src/PortingAssistant.Client/PortingAssistant.Client.CLI.csproj:27]: What's this `NewFolder` for?

**Review by ShanshanHe — APPROVED** [review]
LGTM with minor comment
